package.json

```diff
@@ -34,6 +34,8 @@
     "examples": "webpack-dev-server --config examples/webpack.config.js --no-info --content-base examples",
     "examples-server": "babel-node examples/server.js",
     "flow": "flow check",
+    "format": "prettier --write \"{src,examples}/**/*.js\"",
```

Comment:
I experimented with this, but I couldn't see any different from this vs. `yarn fixlint` -- both apply prettier rules to all the same files. Maybe we can remove this one since the lint one is more specific?

Reply:
Good catch! I agree we can ditch this.